### ISSUE TITLE
Add ./wpt run --verbose as an alias of --log-mach-level debug

### ISSUE THREAD

**Issue by gsnedders** [w3c/web-platform-tests#7846]
It's a pain that we use something non-standard here and not that memorable.

**Comment by qiuzhong** [comment]
If you don't mind, I'll take this issue.

**Comment by qiuzhong** [comment]
If `--verbose` as an alias of `--log-mach-level debug`,  here are my thoughts to handle the options:
* If the command line contains `--verbose` option only, then according to the  alias rule, `log_mach_level` property is set to be `debug`.
* If the command line contains both `--verbose` and `--log-mach-level <whatever level>` options (not recommended), then I guess the attention is to set `log_mach_level` to be the log level value. In such case, `--verbose` is actually invalid and thus has no impact on the `--log-mach-level` option.

@gsnedders , what do you think?

**Comment by jgraham** [comment]
This won't work if implemented naively. If the logging is set to something other than mach then implying the `--mach-log-level` argument will just cause an error about mach logging not being enabled.

**Comment by qiuzhong** [comment]
@gsnedders , do you mean `--verbose` should support all kinds of log formats (`raw, unnitest, xunit, html, tbpl, grouped, errorsummy` from the `mozlog` module code)? My understanding is that besides the `mach` format, it should also support all these formats. In other word, `--verbose` option should be the alias of other `--log_*_level debug` options automatically, not so specific.

Did I misunderstand your intention?

**Comment by qiuzhong** [comment]
@jgraham, please also make a clarification if I misunderstood you.

**Comment by jgraham** [comment]
Yeah, I think you need to find a way to figure out which is going to be the stdout logger and set whatever option is required to make that verbose if it exists. This is probably better done in mozlog itself in https://searchfox.org/mozilla-central/source/testing/mozbase/mozlog/mozlog/commandline.py Note that file has an existing concept of verbose, which I think is different.
